src/vendor_upstream/core/shallowEqual.js

```diff
@@ -1,49 +0,0 @@
-/**
```

Comment:
Unused now, although I'd like to add some shallow diff and reconcile in the reducer to encourage better performance checks.  I can add them using lodash / by hand though.